build.gradle

```diff
@@ -136,7 +136,7 @@ project(':ambry-server') {
                 project(':ambry-utils'),
                 project(':ambry-replication')
         compile "com.codahale.metrics:metrics-core:$metricsVersion"
-        testCompile project(':ambry-router').sourceSets.main.output
+        testCompile project(':ambry-router')
```

Comment:
This fixes an issue with the generated intellij configs. Previously, router code wouldn't be rebuilt when running server tests within the IDE.